x/wasm/client/rest/gov.go

```diff
@@ -28,8 +28,8 @@ type StoreCodeProposalJSONReq struct {
 	// InstantiatePermission to apply on contract creation, optional
 	InstantiatePermission *types.AccessConfig `json:"instantiate_permission" yaml:"instantiate_permission"`
 
-	// PinCode indicates if the code should be pinned as part of the proposal.
-	PinCode bool `json:"pin_code" yaml:"pin_code"`
+	// UnpinCode indicates if the code should not be pinned as part of the proposal.
+	UnpinCode bool `json:"pin_code" yaml:"pin_code"`
```

Comment:
json/yaml comments need to be updated here

Reply:
just noticed this and pushed the change